src/compat/posix/fs/truncate.c

```diff
@@ -0,0 +1,28 @@
+/**
+ * @file
+ *
+ * @brief
+ *
+ * @date 3 Apr 2015
+ * @author Denis Deryugin
+ */
+#include <unistd.h>
+#include <fcntl.h>
+#include <assert.h>
+
+int truncate(const char *path, off_t length) {
+	int fd, ret;
+
+	assert(path);
```

Comment:
It's not a good place for this `assert` in posix function.

Reply:
thanks, done